ckanext/archiver/plugin.py

```diff
@@ -123,7 +123,16 @@ def _is_it_sufficient_change_to_run_archiver(self, package, operation):
 
         # have any resources' url/format changed?
         for res in package.resources:
-            for key in ('url', 'format'):
+            watched_keys = ['format']
```

Comment:
any luck getting a test for this?

Reply:
There are currently no tests for `plugin.py`, so it could be a fair bit of work to implement something.